src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -31,6 +31,15 @@ public static void BufferSize_GettingSameAsWindowSize()
     }
 
     [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.NetcoreUwp)]
+    public static void WindowWidth_WindowHeight_InvalidSize_net46()
+    {
+        Assert.Throws<IOException>(() => Console.WindowWidth = 0);
+        Assert.Throws<IOException>(() => Console.WindowHeight = 0);
+    }
+
+    [Fact]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
We're throwing a different exception than on desktop? Should we instead revert to throwing the same exception rather than changing the test? I agree an argument exception makes more sense, of course.
Cc: @weshaggard